packages/common-ui/lib/list-page/QueryPage.tsx

```diff
@@ -116,18 +123,24 @@ export function QueryPage<TData extends KitsuResource>({
   bulkEditPath,
   reactTableProps,
   defaultSort,
+  selectionMode,
```

Comment:
I recommend adding a default value for the selectionMode

```
selectionMode = false,
```

That way it's never empty. And then the prop can be set as a requirement since a default value will always be provided.